subsurface-mobile-helper.cpp

```diff
@@ -51,6 +54,7 @@ void init_ui()
 
 void run_ui()
 {
+    LOG_STP("run_ui starting");
```

Comment:
white space... everywhere... you really should just update your editor settings :-)

Reply:
I did that a couple of days ago, give me half an hour.

Reply:
no problem - I just find it easier to get things like this fixed before they go into the code :-)

Reply:
i don't think we should merge this in master as is.
isn't this PR only for testing purposes?

Reply:
(a) I find it incredibly painful to build test binaries that are pushed to TestFlight / GooglePlay Beta that are from branches - and it confuses the versioning of those builds
(b) I think it's reasonable to have the code in place to do such timing, even if we disable it for releases